kickstarts/base.ks.erb

```diff
@@ -157,6 +157,8 @@ open-vm-tools
 
 %post --log=/root/anaconda-post.log
 
+yum -y install http://www6.atomicorp.com/channels/atomic/centos/7/x86_64/RPMS/wmi-1.3.14-4.el7.art.x86_64.rpm
```

Comment:
@abellotti Looks like the package we are using for downstream (el6) works fine on a CentOS 7 system, installed and tested it yesterday. Can we just build that into an el7 rpm? (https://brewweb.devel.redhat.com/buildinfo?buildID=425456)